docs/recipes/end2end/smart_city_sdg/workflow_e2e.md

```diff
@@ -252,126 +254,127 @@ python modules/postprocess/postprocess_for_vlm.py \
 
 ## Quickstart (Docker Compose)
 
-1. Clone the repository
+### Clone the Repository
 
-    ```bash
-    git clone https://github.com/NVIDIA/metropolis-sdg-smart-cities.git
-    cd metropolis-sdg-smart-cities
-    ```
+To clone the repository, run the following command:
 
-1. Download sample CARLA logs
+   ```bash
+   git clone https://github.com/NVIDIA/metropolis-sdg-smart-cities.git
+   cd metropolis-sdg-smart-cities
+   ```
 
-    > **Note:** Sample logs are provided by Inverted AI. Please review the data [terms of use](https://github.com/inverted-ai/metropolis/blob/master/LICENSE.md) to determine whether they are appropriate for your purposes. If you have your own data you may skip this step and place it under `./data/examples/`
+### Download the Sample CARLA Logs
 
-    ```bash
-    git clone https://github.com/inverted-ai/metropolis.git
-    mv ./metropolis/examples ./data/examples
-    ```
+To download sample CARLA logs, run the following command:
 
-1. Set up the deployment configuration.
+> **Note:** Sample logs are provided by Inverted AI. Review the data [terms of use](https://github.com/inverted-ai/metropolis/blob/master/LICENSE.md) to determine whether they are appropriate for your purposes. If you have your own data, you can skip this step and place it under `./data/examples/`
 
-    You need to provide your NGC_API_KEY [with access to pull images from build.nvidia](https://build.nvidia.com/settings/api-keys) and Hugging Face Token with access to the checkpoints mentioned under [Prerequisites](#prerequisites). The other parameters are optional to configure GPU IDs that each NIM/service should run on, and ports to launch the NIMs on. By default, they assume a homogeneous deployment to a system with at least 4x RTX 6000 Pro or equivalent.
+```bash
+git clone https://github.com/inverted-ai/metropolis.git
+mv ./metropolis/examples ./data/examples
+```
 
-    ```bash
-    cd deploy/compose
-    cp env.example env
-    # Edit values for NGC_API_KEY, HF_TOKEN, GPU IDs, ports, etc.
-    ```
+### Set Up the Deployment Configuration
 
-1. Deploy the stack.
+You need to provide your NGC_API_KEY [with access to pull images from build.nvidia](https://build.nvidia.com/settings/api-keys) and Hugging Face Token with access to the checkpoints mentioned in [Prerequisites](#prerequisites). The other parameters are optional to configure the GPU IDs that each NIM/service should run on and ports to launch the NIMs on. By default, they assume a homogeneous deployment to a system with at least 4x RTX 6000 Pro or equivalent.
```

Comment:
What "assumes" here? Is it the "other parameters"? If so, it's not clear how the assumption is built in to the parameter. I suggest we reword here to clarify.